tests/integration/pilot/gateway_test.go

```diff
@@ -147,51 +147,138 @@ func ManagedGatewayTest(t framework.TestContext) {
 	t.ConfigIstio().YAML(apps.Namespace.Name(), `apiVersion: gateway.networking.k8s.io/v1beta1
 kind: Gateway
 metadata:
-  name: gateway
+  name: gateway-1
 spec:
   gatewayClassName: istio
   listeners:
   - name: default
-    hostname: "*.example.com"
+    hostname: "*.example-1.com"
     port: 80
     protocol: HTTP
 ---
 apiVersion: gateway.networking.k8s.io/v1beta1
 kind: HTTPRoute
 metadata:
-  name: http
+  name: http-1
 spec:
   parentRefs:
-  - name: gateway
+  - name: gateway-1
   rules:
   - backendRefs:
     - name: b
       port: 80
+---
+apiVersion: gateway.networking.k8s.io/v1beta1
```

Comment:
why do we need 2 gateways to test dual stack?

Reply:
Replaced with single gateway and two HTTPRoutes